docs/public-offer/index.rst

```diff
@@ -241,5 +241,17 @@ To update AMI product listing with multiple requests for product details (Descri
       --config listing_configuration.yaml
 
 
+Download AMI product listing details
+------------------------------------
+
+To download AMI product listing information as a YAML file, run the command below, passing the product ID and product configuration file:
```

Comment:
this could be more clear on what the expected outcome is. to me, this reads as if you need to supply an existing awsmp config. it would be more clear is we spell out that `--config` is the output path